### PR TITLE
Modal v2 changes

### PR DESCRIPTION
**Pull request by roscioli**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pinwheel/react-modal",
-  "version": "1.0.6",
+  "version": "2.0.0-alpha",
```
Comment (roscioli) [inline package.json:3]: Calling this alpha so we can test it before a full release

File: src/index.tsx [hunk 2]
```diff
@@ -1,19 +1,45 @@
...
+  | 'invalidUserInput'
+  | 'invalidLinkToken'
+
+export type Error = {
+  type: ErrorType
+  code: string
+  message: string
+}
+
+type EventPayload =
```
Comment (TrekSoft) [inline src/index.tsx:26]: Nit: I think we can remove this - the payloads are defined on the Link side and once you add {} as a possible type, it allows anything compile time anyway.

**Comment by TrekSoft** [comment]
Do you have to update it somewhere to point to the new Link V2 or Portal V2 code?

File: src/index.tsx (hunk 2 above)
Comment (roscioli) [reply src/index.tsx:26]: These types are to help users of our package know the types. Their IDE will pick these type definitions up.

**Comment by roscioli** [comment]
> Do you have to update it somewhere to point to the new Link V2 or Portal V2 code?

Oh yah probably lol

**Pull request merged**